{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -93,7 +93,7 @@ def build_artifacts(
     import vivarium_cluster_tools as vct
     vct.mkdir(output_dir, parents=True, exists_ok=True)
 
-    check_for_existing(output_dir, location, append, replace_keys)
+    check_for_existing(Path(output_dir), location, append, replace_keys)
```

Comment:
I think it'd be better if you defined it as `output_dir = Path(output_dir)` above line 94 so that each usage of `output_dir` uses a `Path` rather than a `str`. That's how it was done in CIFF